src/components/navbar/navbar.component.tsx

```diff
@@ -93,7 +99,44 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="Panel"
+                  aria-labelledby="Panel Icon"
+                  name="Panel"
+                  isCollapsible
+                  onClick={onClickSideNavExpand}
+                  isActive={isSideNavExpanded}
+                >
+                  <Switcher20 />
+                </HeaderGlobalAction>
               </HeaderGlobalBar>
+              <HeaderPanel aria-label="Header Panel" expanded>
+                <Switcher aria-label="Switcher Container">
```

Comment:
All of these are hardcoded, but should come from microfrontends!

(Also note that you should *never* use absolute URLs - especially not such ones that are obviously invalid).

Reply:
How can I go about this please? I actually thought of it.

Reply:
First: What do you want to achieve? I see no structure in this PR, no good description.

Second: You create navigation elements here that point to code living in some microfrontends. This is a no go. You should have/use a **generic** extension slot that will be utilized by the microfrontends referenced from the links. This way, the links are

- configurable
- only available if the referenced microfrontend is loaded
- does not use knowledge that is not owned by the microfrontend (thus being more consistent and easier to maintain)

Reply:
Hi @FlorianRappl Thanks again for reviewing my pull requests.

I was working on this [pull request](https://issues.openmrs.org/browse/MF-430). The task was to add an app-nav button and a dropdown with options driven from extension slots. I checked the design on extension slots and it showed these options: Clinical data, patients, schedules, reports. I didn't know exactly how to fit in those options, that's why I used absolute urls.

Reply:
Hi @FlorianRappl Thanks again for reviewing my pull requests.

I was working on this [Issue](https://issues.openmrs.org/browse/MF-430). The task was to add an app-nav button and a drop-down with options driven from extension slots. I checked the design on extension slots and it showed these options: Clinical data, patients, schedules, reports. I didn't know exactly how to fit in those options, that's why I used absolute urls.

Reply:
Again, the PR in this form is not acceptable.